plugin/Installer/DatabaseChanges.php

```diff
@@ -296,19 +297,27 @@ private function installDataQueryForms()
 				$template = $templatePathsForm;
 			}
 		}
-		$this->_pWPDB->insert(
-			$tableName,
```

Comment:
This attempt to make the default form better is nice, but the default form will have to be removed in another project, so I think this block should be reverted.

Reply:
Ok, I rollback code, but it's will make failed run test

Reply:
If you check `TestClassDatabaseChanges`, methods `testInstall()` and `testInstallMigrationsDataSimilarEstates()`, there's a filter that catches `CREATE TABLE` queries. A similar one would be needed for `INSERT` queries.